docs/api/index.md

```diff
@@ -1249,71 +1691,179 @@ Note that `trigger` accepts a second argument to pass options to the triggered E
 await wrapper.trigger('keydown', { keyCode: 65 })
 ```
 
-### `unmount`
+::: tip
+You should use `await` when you call `trigger` to ensure that Vue updates the DOM before you make an assertion.
+:::
+
+### unmount
+
+Unmount the application from the DOM.
+
+**Signature:**
+
+```ts
+unmount(): void
+```
+
+**Details:**
 
-Unmount the application from the DOM via Vue's `unmount` method. Only works on the root `VueWrapper` returned from `mount`. Useful for manual clean-up after tests.
+It only works on the root `VueWrapper` returned from `mount`. Useful for manual clean-up after tests.
 
 `Component.vue`:
 
 ```vue
 <template>
   <div />
 </template>
+
+<>
+export default {
+  unmounted() {
+    console.log('unmounted!')
+  }
+}
+</>
 ```
 
 `Component.spec.js`:
 
 ```js
+import { mount } from '@vue/test-utils'
+import Component from './Component.vue'
+
 test('unmount', () => {
   const wrapper = mount(Component)
 
-  wrapper.unmount() // removed from DOM
+  wrapper.unmount()
+  // Component is removed from DOM.
+  // console.log has been called with 'unmounted!'
 })
 ```
 
 ## Wrapper properties
 
-### `vm`
+### vm
 
-This is the `Vue` instance. You can access all of the [instance methods and properties of a vm](https://v3.vuejs.org/api/instance-properties.html) with `wrapper.vm`. This only exists on `VueWrapper`.
+**Signature:**
 
-## Global Config
+```ts
+vm: ComponentPublicInstance
+```
 
-### `config.global`
+**Details:**
 
-Instead of configuring global mounting options on a per-test basis, you can configure them globally. These will be used by default every time you `mount` a component. You can override the defaults by via mounting options.
+The `Vue` app instance. You can access all of the [instance methods](https://v3.vuejs.org/api/instance-methods.html) and [instance properties](https://v3.vuejs.org/api/instance-properties.html).
```

Comment:
Like with the props section, should there be a warning here saying test against dom elements whenever is possible eg click on them and see the result, as opposed to call a method on the `vm`

Reply:
sure! done. nice suggestion